ember-server/shared/src/main/scala/org/http4s/ember/server/internal/ServerHelpers.scala

```diff
@@ -485,8 +494,12 @@ private[server] object ServerHelpers extends ServerHelpersPlatform {
                   EmberException.ReadTimeout(_) =>
                 Applicative[F].pure(None)
               case err =>
-                errorHandler(err)
-                  .handleError(_ => serverFailure.covary[F])
+                (err match {
+                  case err: Parser.Request.ReqPrelude.ParsePreludeError =>
+                    requestLineParseErrorHandler(err)
```

Comment:
Might be dumb question: doesn't the request prelude involve more than the request line?

Reply:
https://github.com/http4s/http4s/blob/6931-special-case-handle-request-line-parse-failure/ember-core/shared/src/main/scala/org/http4s/ember/core/Parser.scala#L198

I don't think so. Method, Http Version, URI.

Reply:
Ah, thanks. So then maybe `ReqPrelude` should be `ReqLine` if that's the "official" name?